src/main/java/com/enderio/base/common/util/EntityCaptureUtils.java

```diff
@@ -55,13 +55,13 @@ public Component errorMessage() {
      */
     public static CapturableStatus getCapturableStatus(EntityType<?> type, @Nullable Entity entity) {
 
-        if (entity != null && isBlacklistedBoss(entity))
+        if (entity != null && isBlacklistedBoss(entity)) //Do we keep this special case?
```

Comment:
What would the case be for removing this? If I'm thinking about this correctly, this check will have to remain as the soul vial will not be using soul data, right?

Reply:
No, but I did add a blacklist tag for the soul vials as well, since I also added one for the spawner. That way other mobs can be blacklisted using the tag so this check isn't needed. There also is a bosses tag now to replace this method I think?

Reply:
Wait; so there's a tag to define what mobs are bosses/blacklisted and we check that now? If so then thats perfectly fine :)

Reply:
Yes forge added that tag some time back. I made a blacklist tag that has the forge bosses tag inside of it. So for the soul vials and spawners, both have a single tag (which has bosses as well).